st2common/st2common/models/db/execution.py

```diff
@@ -178,6 +179,7 @@ class ActionExecutionOutputDB(stormbase.StormFoundationDB):
     runner_ref = me.StringField(required=True)
     timestamp = ComplexDateTimeField(required=True, default=date_utils.get_datetime_utc_now)
     output_type = me.StringField(required=True, default='output')
+    delay = me.IntField(required=True, default=0)
```

Comment:
Actually, looking at the code, I don't think we even need this attribute on this model - ``ActionExecutionOutputDB``.